Client/Frontend/Library/LibraryViewController/LibraryViewController.swift

```diff
@@ -47,38 +40,67 @@ class LibraryViewController: UIViewController {
+        toolbar.setItems([UIBarButtonItem(customView: librarySegmentControl)], animated: false)
+
+        return toolbar
     }()
 
-    fileprivate lazy var topLeftButton: UIButton =  {
-        let button = UIButton()
-        button.setTitle(Strings.BackTitle, for: .normal)
-        button.titleLabel?.font = UIFont.systemFont(ofSize: 17, weight: .regular)
-        button.setTitleColor(UIColor.systemBlue, for: .normal)
-        button.titleLabel?.textAlignment = .left
-        button.addTarget(self, action: #selector(topLeftButtonAction), for: .touchUpInside)
+    fileprivate lazy var topLeftButton: UIBarButtonItem =  {
+        let button = UIBarButtonItem(image: UIImage.templateImageNamed("goBack"),
+                                     style: .plain,
+                                     target: self,
+                                     action: #selector(topLeftButtonAction))
+        button.accessibilityIdentifier = "libraryPanelTopLeftButton"
         return button
     }()
-    
-    fileprivate lazy var topRightButton: UIButton =  {
-        let button = UIButton()
-        button.setTitle(String.AppSettingsDone, for: .normal)
-        button.titleLabel?.font = UIFont.systemFont(ofSize: 17, weight: .regular)
-        button.setTitleColor(UIColor.systemBlue, for: .normal)
-        button.addTarget(self, action: #selector(topRightButtonAction), for: .touchUpInside)
+
+    fileprivate lazy var topRightButton: UIBarButtonItem =  {
+        let button = UIBarButtonItem(title: String.AppSettingsDone,
+                                     style: .done,
+                                     target: self,
+                                     action: #selector(topRightButtonAction))
+        button.accessibilityIdentifier = "libraryPanelTopRightButton"
         return button
     }()
 
-    init(profile: Profile) {
-        self.profile = profile
+    fileprivate lazy var bottomLeftButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(image: UIImage.templateImageNamed("nav-add"),
+                                     style: .plain,
+                                     target: self,
+                                     action: #selector(bottomLeftButtonAction))
+        button.accessibilityIdentifier = "libraryPanelBottomLeftButton"
+        return button
+    }()
+
+    fileprivate lazy var bottomRightButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(title: Strings.BookmarksEdit,
+                                     style: .plain,
+                                     target: self,
+                                     action: #selector(bottomRightButtonAction))
+        button.accessibilityIdentifier = "bookmarksPanelBottomRightButton"
+        return button
+    }()
+
+    lazy var flexibleSpace: UIBarButtonItem = {
+        return UIBarButtonItem(barButtonSystemItem: .flexibleSpace,
+                               target: nil,
+                               action: nil)
+    }()
```

Comment:
Same styling nit

```suggestion
    fileprivate lazy var bottomLeftButton: UIBarButtonItem = {
        let button = UIBarButtonItem(image: UIImage.templateImageNamed("nav-add"), style: .plain, target: self,  action: #selector(bottomLeftButtonAction))
        button.accessibilityIdentifier = "libraryPanelBottomLeftButton"
        return button
    }()

    fileprivate lazy var bottomRightButton: UIBarButtonItem = {
        let button = UIBarButtonItem(title: Strings.BookmarksEdit, style: .plain, target: self, action: #selector(bottomRightButtonAction))
        button.accessibilityIdentifier = "bookmarksPanelBottomRightButton"
        return button
    }()

    lazy var flexibleSpace: UIBarButtonItem = {
        return UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: nil, action: nil)
    }()
```

Reply:
Is the newline style not easier to read, though? That's why I've always done it.

Reply:
@electricRGB yeah, plus XCode has built in wrapping so it becomes easier to read